simple_rl/agents/func_approx/dsc/SkillChainingAgentClass.py

```diff
@@ -102,10 +103,10 @@ def __init__(self, mdp, max_steps, lr_actor, lr_critic, ddpg_batch_size, device,
 		# We use (double-deep) (intra-option) Q-learning to learn the Q-values of *options* at any queried state Q(s, o)
 		# We start with this DQN Agent only predicting Q-values for taking the global_option, but as we learn new
 		# options, this agent will predict Q-values for them as well
+		print("this is state space size" + str(self.mdp.state_space_size()))
 		self.agent_over_options = DQNAgent(self.mdp.state_space_size(), 1, trained_options=self.trained_options,
 										   seed=seed, lr=1e-4, name="GlobalDQN", eps_start=1.0, tensor_log=tensor_log,
-										   use_double_dqn=True, writer=self.writer, device=self.device,
-										   pixel_observation=True)
+										   use_double_dqn=True, writer=self.writer, device=self.device)
```

Comment:
Here, can we say `pixel_observation=self.mdp.pixel_observation` or as an argument to the DSC class